### DISCUSSION TITLE
[Question]: Why LibreChat creates these orcexec.* files?

### DISCUSSION THREAD

**Discussion by gafda** [danny-avila/LibreChat#1742] [Q&A] [answered]
### What is your question?

My firewall capture unusual behavior within the container of LibreChat v0.6.6 stating:

```text
* Process name: node
* Process binary fullpath: /usr/local/bin/node
* Library name: /root/orcexec.aibHLp (deleted)

This finding poses a high risk because it indicates that an attacker may have access to the system and is able to execute arbitrary code.
```

What is this exectly? Why does it create this? Can this be avoided?

### More Details

My instance is completely vanilla with a single connection to AzureOpenAI. I haven't added plugins or any other stuff...

### What is the main subject of your question?

Other

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Seems like an artifact from running the node:alpine linux container: https://github.com/lovell/sharp/issues/1039

I would not worry about it since this docker image has no vulnerabilities but there are instructions on how to disable it in the link I sent: 
![image](https://github.com/danny-avila/LibreChat/assets/110412045/36799510-7729-4800-86eb-ebb30756fa40)

**Reply by danny-avila** [reply, marked as answer]
more info: https://github.com/GStreamer/orc

It is likely used by the sharp package under a specific linux env, makes sense they would report this as well since we use the sharp package in this project.
